common/test/acceptance/pages/lms/instructor_dashboard.py

```diff
@@ -452,6 +456,109 @@ def is_cohorted(self, state):
         if state != self.is_cohorted:
             self.q(css=self._bounded_selector('.cohorts-state')).first.click()
 
+    def toggle_discussion_topics(self):
+        """
+        Shows the discussion topics.
```

Comment:
I'd change this to say "Toggles showing of discussion topics", since it doesn't always show them.